package.json

```diff
@@ -64,12 +64,17 @@
   },
   "devDependencies": {
     "@edx/stylelint-config-edx": "1.1.0",
+    "babel-jest": "^23.0.1",
```

Comment:
I'm not sure what's causing the issues with the python tests, they seem to be coming from the cherry-picked change from #18364 not really sure how to fix that, some help there would be appreciated.

I've made linting fixes to the code. I also did ran eslint to automatically fix some of the issues it can in the other code. It's in a separate commit so I can remove that if you feel it's a bit much for this PR.